testsuite/module-publish/src/packages/simple/sources/Simple.move

```diff
@@ -104,19 +124,25 @@ module 0xABCD::simple {
         counter.count;
     }
 
-    public entry fun step_destination(owner: &signer, destination: address) acquires Counter {
-        let value = {
-            assert!(exists<Counter>(destination), error::invalid_argument(ECOUNTER_RESOURCE_NOT_PRESENT));
+    public entry fun step_destination(_owner: &signer, destination: address) acquires Counter {
+        assert!(exists<Counter>(destination), error::invalid_argument(ECOUNTER_RESOURCE_NOT_PRESENT));
+        let counter = borrow_global_mut<Counter>(destination);
+        *(&mut counter.count) = counter.count + COUNTER_STEP;
+    }
 
-            let counter = borrow_global_mut<Counter>(destination);
-            *(&mut counter.count) = counter.count + COUNTER_STEP;
-            counter.count
-        };
-        if (exists<Counter>(signer::address_of(owner))) {
-            let counter_owner = borrow_global_mut<Counter>(signer::address_of(owner));
-            *(&mut counter_owner.count) = value;
+    public entry fun step_destination_agg_v2(_owner: &signer, destination: address) acquires CounterAggV2 {
+        assert!(exists<CounterAggV2>(destination), error::invalid_argument(ECOUNTER_RESOURCE_NOT_PRESENT));
+        let counter = borrow_global_mut<CounterAggV2>(destination);
```

Comment:
Same here: why not do `let counter = &mut borrow_global_mut<CounterAggV2>(destination).count;` so that then we `aggregator_v2::try_add` on a variable only?

Reply:
this seems cleaner to me?